timeseries/src/autogluon/timeseries/models/multi_window/multi_window_model.py

```diff
@@ -238,32 +244,35 @@ def save(self, path: str = None, verbose=True) -> str:
             most_recent_model.save()
         return save_path
 
-    def persist(self):
+    def persist(self) -> AbstractTimeSeriesModel:
         if self.most_recent_model is None:
             raise ValueError(f"{self.name} must be fit before persisting")
-        self.most_recent_model.persist()
+        return self.most_recent_model.persist()
```

Comment:
Should this method return `self` instead of `self.most_recent_model` (for consistency with `AbstractTimeSeriesModel.persist`)?
```suggestion
    def persist(self) -> Self:
        if self.most_recent_model is None:
            raise ValueError(f"{self.name} must be fit before persisting")
        self.most_recent_model.persist()
        return self
```